doc/CODING.md

```diff
@@ -132,6 +132,72 @@ So do not give this responsibility out of hands entirely.
 
 **Example:** [src/libs/elektra/kdb.c](/src/libs/elektra/kdb.c)
 
+#### Clang Format
+
+To guarantee consistent formatting we use [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html) (version `6.0` or later) to format all C and C++ code in the repository. Since our build server also check the style for every pull request you might want to make sure you reformat your C/C++ code changes with this tool.
```

Comment:
Maybe also mention where to find out which version of clang-format the build server uses